packages/transactions/src/configs.ts

```diff
@@ -6,9 +6,6 @@ export const CONTRACT_MAX_SIZE = 16 * 1024;
 /** Maximum number of witnesses. */
 export const MAX_WITNESSES = 16;
 
-/** Maximum gas per transaction. */
-export const MAX_GAS_PER_TX = bn(10_000_000);
```

Comment:
Looks like you purged the constant `MAX_GAS_PER_TX` but the PR name and the changeset mentions `MAX_GAS_PER_PREDICATE` instead

Reply:
You're right :man_facepalming: 

Fixed